### ISSUE TITLE
Status of the project

### ISSUE THREAD

**Issue by fbeaulier** [Daparrag/Opus_CotexM4_microcontroller#1]
Hello Diego,
I'm interrested in this project, i would like to benchmark libopus on a cortex-M7 MCU.
Can you please tell me the status of the project ?
What have you done and what is missing ?
Have you been able to make it run on a cortex-M4 ?
How do you build for cortex-M4 ? 

Thank's for any help you can provide

Francois

**Comment by Daparrag** [comment]
hope is not to late. 

are you still interesting?

**Comment by Daparrag** [comment]
Can you please tell me the status of the project ?
-  I completed this project 2 years ago. it was working at that time as I think it will works now. 
What have you done and what is missing ?
-  I added support for cortex-M4 micro-controllers. Missing nothing.
Have you been able to make it run on a cortex-M?
- Yes it works for cortex-M4 mics

How do you build for cortex-M4 ?
- you have to cross compile it with arm-v7 compilers.

**Comment by fbeaulier** [comment]
Yes too late but it was too big anyway, i needed something small and simple, i used ADPCM.
